### PR TITLE
Added ability to set additional feedback messages in assessment mode

### PR DESCRIPTION
**Pull request by e-kolpakov**

### CODE REVIEW

File: mentoring/mentoring.py [hunk 1]
```diff
@@ -348,6 +352,7 @@ def handleAssessmentSubmit(self, submissions, suffix):
                     'score_type': 'proficiency',
                 })
                 event_data['final_grade'] = score.raw
+                message = self.get_message_html('on-assessment-review')
```
Comment (bradenmacdonald) [inline mentoring/mentoring.py]: Can't you just assign `self.assessment_message` here instead of calling `get_message_html`?

**Comment by bradenmacdonald** [comment]
Changes look good, but you need to add an example of the new message to the tests and Readme file.

**Comment by bradenmacdonald** [comment]
Actually, I was unable to test the jquery-related changes in Apros because this doesn't seem to work with HTML messages.

I added this to a mentoring block's XML:

```xml
    <message type="on-assessment-review">
      <html>Testing!!! Try going to the <a href="/jump_to_id/f5d414b62d3146d98f950cf067fd5db6 ">Mentoring Test Unit</a>.</html>
    </message>
```

But then in both Studio and Apros, the link has been stripped out and all I get is:

```html
<script type='text/template' id='light-child-template'>
Testing!!! Try going to the
</script>
```

**Comment by mtyaka** [comment]
It works fine for me in apros.

**Comment by mtyaka** [comment]
Other than Braden's comments this looks good to me :+1:

**Comment by e-kolpakov** [comment]
@bradenmacdonald try wrapping links in `<p>` - likely a bug with `html` light child. Looks like if it gets a text node at the top it ignores everything else.

**Comment by e-kolpakov** [comment]
@bradenmacdonald Added tests and readme entry

**Comment by bradenmacdonald** [comment]
:+1:

**Pull request merged**